### PR TITLE
#164198135 Set up empty react project

### PR DESCRIPTION
**Pull request by thislekan**
#### What does this PR do?

- set up an empty react project

#### Description of Task to be completed?

- add react router
- add sass processors
- improve the production config
- set up components

#### How should this be manually tested?

If you don't have ```node.js``` installed, please follow this link to [download node.js](https://nodejs.org/en/)

Download the repo by clicking the green ```clone or download``` button above. If you chose the download option, unzip your files into a folder. If you chose the clone option, then run ```git clone 'the-url-copied-by-clicking-the-green-button'```

Please note: This information assumes a basic understanding of node and git

> Run the following commands in your terminal:

```
# Change directory to clone or zipped folder
cd apollo-ah-frontend (or the name of the folder you zipped the files in)

# Switch to staging branch
git checkout staging
git checkout staging/164198135/set-up-react

# Install package modules
npm install

# Start up the server
npm run start:dev

```

#### What are the relevant pivotal tracker stories?

- [#164198135](https://www.pivotaltracker.com/story/show/164198135)

### CODE REVIEW

**Review by aanchirinah — DISMISSED** [review]
LGTM

File: webpack.config.prod.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/* eslint-disable react/wrap-multilines */
+const path = require('path');
+const HtmlWebPackPlugin = require('html-webpack-plugin');
```
Comment (aanchirinah) [inline webpack.config.prod.js]: #### Nitpick
- Are you performing minimization in this file?

File: webpack.config.prod.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+/* eslint-disable react/wrap-multilines */
+const path = require('path');
```
Comment (aanchirinah) [inline webpack.config.prod.js:2]: #### Nitpick
- Are you performing minimization in this file?

**Review by aanchirinah — CHANGES_REQUESTED** [review]
Good job, kindly attend to my requests.

File: webpack.config.prod.js (hunk 2 above)
Comment (thislekan) [reply webpack.config.prod.js:2]: @aanchirinah Webpack is doing minification by default. So it isn't done in this config but in the build script in `package.json`

Comment (aanchirinah) [reply webpack.config.prod.js:2]: I don't see any `UglifyJSPlugin` in your package.json, @thislekan , can you explicitly tell me Webpack 4 is doing minification properly without `UglifyJSPlugin` installed?

Comment (thislekan) [reply webpack.config.prod.js:2]: @aanchirinah this is a link to how minification is done by default in production [Webpack 4 minification](https://webpack.js.org/guides/production#minification)

Comment (aanchirinah) [reply webpack.config.prod.js:2]: @thislekan , okay thanks for the update , I'm still old school webpack 1.

**Review by aanchirinah — APPROVED** [review]

**Review by davdwhyte87 — APPROVED** [review]
LGTM

**Pull request merged**